### DISCUSSION TITLE
Deploying a smart contract throws an error and unable to continue.

### DISCUSSION THREAD

**Discussion by sumi0820** [WalletConnect/walletconnect-monorepo#4018] [Developer Support] [answered]
I'm developing Dapp(wallet) with walletConnect, and trying to deploy a smart contract to testnet. Using Ledger Live via walletConnect,   it can be connected and I can get a session, but I get the attached screen and cannot press continue when trying to deploy the contract.     
![Screenshot 2023-12-07 at 10 15 26](https://github.com/WalletConnect/walletconnect-monorepo/assets/48553998/ec609cb4-ee75-4b53-ab90-56e87900c11a)

Although I input the amount as below, it throws the same screen. I'm not sure if this can be controlled by walletConnect or Ledger Live end, but it'd be appreciated if anyone would share the solution.   

```
        import { useRequest } from '@walletconnect/modal-sign-react';
        const { request } = useRequest();

        const response = await request({
          topic: session !== undefined ? session : '',
          chainId: 'eip155:5',
          jsonrpc: '2.0',
          request: {
            method: 'eth_sendTransaction',
            params: [
              {
                from: signerWalletAddress,
                to,
                data,
                nonce,
                gasPrice: '0xbb5e',
                gas: '0x5208',
                value: '0x00',
                amount: '23',
              },
            ],
          },
        });
```

**Comment by glitch-txs** [comment, marked as answer]
amount should be in wei
